.drone.yml

```diff
@@ -25,6 +25,21 @@ pipeline:
     when:
       branch: master
 
+  publish_nightly:
+    image: lampepfl/dotty:24-04-2017
+    pull: true
+    commands:
+      - ./project/scripts/sbt ";clean ;publishLocal" "${CI_PUBLISH}"
+      - ./project/scripts/sbt "dotty-sbt-bridge/scripted source-dependencies/*1of3" "${CI_PUBLISH}"
```

Comment:
Note that currently the source-dependencies test are in sbt-scripted-tests, not dotty-sbt-bridge, but we could move them there.